subsys/bluetooth/host/gatt.c

```diff
@@ -2482,11 +2482,22 @@ static void sc_restore_rsp(struct bt_conn *conn,
 	 * The client receives and confirms a Service Changed indication.
 	 */
 	cfg = find_cf_cfg(conn);
+
```

Comment:
Seems like an unrelated change? The general style in this subsystem is not to have an empty line between assignment to a variable and testing the variable’s value.

Reply:
Fixed